Gruntfile.js

```diff
@@ -260,7 +260,7 @@ module.exports = function( grunt ) {
 					'echo "Generating contributor list since <%= fromDate %>"',
 					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce --fromDate <%= fromDate %>' +
 					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format md --sortOrder desc' +
-					' --showlogin true > contributors.md'
+					' --showlogin true --sha <%= sha %> > contributors.md'
```

Comment:
We could include `--filter renovate-bot` to remove renovate bot, at least @peterfabian did it before.